### PR TITLE
Task/remove default export

### PR DESCRIPTION
**Pull request by v0ldemar01**

### CODE REVIEW

File: client/src/components/common/common.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-export { default as Segment } from './segment/segment.jsx';
-export { default as Message } from './message/message.jsx';
-export { default as Spinner } from './spinner/spinner.jsx';
-export { default as Checkbox } from './checkbox/checkbox.jsx';
-export { default as IconButton } from './icon-button/icon-button.jsx';
-export { default as PublicRoute } from './public-route/public-route.jsx';
-export { default as PrivateRoute } from './private-route/private-route.jsx';
-export { default as Notifications } from './notifications/notifications.jsx';
-export { default as CopyBufferInput } from './copy-buffer-input/copy-buffer-input.jsx';
+export * from './portal/portal.jsx';
```
Comment (what1s1ove) [inline client/src/components/common/common.js:2]:
```suggestion
export { Portal } from './portal/portal.jsx';
```
export in this way

File: client/src/components/app.jsx [hunk 2]
```diff
@@ -10,11 +10,11 @@ import {
...
-import SignPage from 'components/sign/sign';
-import NotFoundPage from 'components/not-found/not-found';
-import ProfilePage from 'components/profile/profile';
-import SharedPostPage from 'components/shared-post/shared-post';
-import ThreadPage from 'components/thread/thread';
+import { Login as SignPage } from 'components/sign/sign';
+import { NotFound as NotFoundPage } from 'components/not-found/not-found';
+import { Profile as ProfilePage } from 'components/profile/profile';
+import { SharedPost as SharedPostPage } from 'components/shared-post/shared-post';
+import { Thread as ThreadPage } from 'components/thread/thread';
```
Comment (what1s1ove) [inline client/src/components/app.jsx:17]:
```suggestion
import { Sign } from 'components/sign/sign';
import { NotFound } from 'components/not-found/not-found';
import { Profile } from 'components/profile/profile';
import { SharedPost } from 'components/shared-post/shared-post';
import { Thread } from 'components/thread/thread';
```
should be the same as folder name

File: client/src/components/sign/components/components.js [hunk 3]
```diff
@@ -1,4 +1,2 @@
-import LoginForm from './login-form/login-form.jsx';
-import RegistrationForm from './registration-form/registration-form.jsx';
-
-export { LoginForm, RegistrationForm };
+export * from './login-form/login-form.jsx';
+export * from './registration-form/registration-form.jsx';
```
Comment (what1s1ove) [inline client/src/components/sign/components/components.js:2]:
```suggestion
export { LoginForm } from './login-form/login-form.jsx';
export { RegistrationForm } from './registration-form/registration-form.jsx';
```

File: client/src/components/thread/components/components.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import AddPost from './add-post/add-post.jsx';
-import ExpandedPost from './expanded-post/expanded-post.jsx';
-import SharedPostLink from './shared-post-link/shared-post-link.jsx';
-
-export { AddPost, ExpandedPost, SharedPostLink };
+export * from './comment/comment.jsx';
```
Comment (what1s1ove) [inline client/src/components/thread/components/components.js:1]: specify names

File: client/src/index.jsx [hunk 5]
```diff
@@ -2,8 +2,8 @@ import { StrictMode } from 'react';
 import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
-import App from 'components/app';
-import store from 'store/store';
+import { Routing as App } from 'components/app';
```
Comment (what1s1ove) [inline client/src/index.jsx:5]:
```suggestion
import { App } from 'components/app';
```
should be the same as folder name

File: server/knexfile.js [hunk 6]
```diff
@@ -31,4 +31,4 @@ const knexConfig = {
   ...knexSnakeCaseMappers({ underscoreBetweenUppercaseLetters: true })
 };
 
-export default knexConfig;
+export { knexConfig };
```
Comment (what1s1ove) [inline server/knexfile.js:34]: this file should be with a default export
add exclude to the backend eslint file https://github.com/BinaryStudioAcademy/bsa-winter-2021-2022-bws/blob/development/backend/.eslintrc.yml#L11

File: server/src/data/models/index.js [hunk 7]
```diff
@@ -1,5 +1,5 @@
-export { default as CommentModel } from './comment/comment.model.js';
-export { default as ImageModel } from './image/image.model.js';
-export { default as PostModel } from './post/post.model.js';
-export { default as PostReactionModel } from './post-reaction/post-reaction.model.js';
-export { default as UserModel } from './user/user.model.js';
+export * from './post/post.model.js';
```
Comment (what1s1ove) [inline server/src/data/models/index.js:1]: specify names

File: server/src/data/models/index.js [hunk 8]
```diff
@@ -1,5 +1,5 @@
-export { default as CommentModel } from './comment/comment.model.js';
```
Comment (what1s1ove) [inline server/src/data/models/index.js:1]: lets rename this file to `server/src/data/models/models.js`

**Review by what1s1ove — COMMENTED** [review]
```
  import/no-default-export:
    - error
```
add this to the root eslint

File: client/src/components/common/common.js (hunk 1 above)
Comment (v0ldemar01) [reply client/src/components/common/common.js:2]: @what1s1ove This approach related to only jsx components?

Comment (what1s1ove) [reply client/src/components/common/common.js:2]: @v0ldemar01 not, actually we want to remove "star" export/export

Comment (v0ldemar01) [reply client/src/components/common/common.js:2]: I see, but as for this image, stay as prev?
image

Comment (what1s1ove) [reply client/src/components/common/common.js:2]: with the old one we will do it in the separate ticket

File: server/src/data/models/abstract/abstract.model.js [hunk 9]
```diff
@@ -1,6 +1,6 @@
 import { Model } from 'objection';
 
-class Abstract extends Model {
+class AbstractModel extends Model {
```
Comment (what1s1ove) [inline server/src/data/models/abstract/abstract.model.js:3]: we already have `model` in the file name. So we need to add it to the class name. Rename it as { Abstract as AbstractMobel } in places where you need it. Should be fixed for all models

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: server/src/data/models/abstract/abstract.model.js (hunk 9 above)
Comment (v0ldemar01) [reply server/src/data/models/abstract/abstract.model.js:3]: @what1s1ove you mean rename in current file like on screen or at import in different modules?
image

Comment (what1s1ove) [reply server/src/data/models/abstract/abstract.model.js:3]: **at import in different modules** 👍

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**